lib/pharos/phases/migrate_worker.rb

```diff
@@ -32,13 +32,18 @@ def migrate_1_3?
       end
 
       def migrate_1_3
-        logger.info { 'Upgrade kubelet config for 1.10' }
+        logger.info { 'Upgrade kubelet config' }
 
         # use the new version of kubeadm to write out /var/lib/kubelet/config.yaml for new kubelet version to be installed
         # the kube master must be running, which is the case for upgrades
         host_configurer.upgrade_kubeadm(Pharos::KUBEADM_VERSION)
 
         @ssh.exec!("sudo /usr/local/bin/pharos-kubeadm-#{Pharos::KUBEADM_VERSION} upgrade node config --kubelet-version=v#{Pharos::KUBE_VERSION}")
+
+        kubeadm_flags = @ssh.file("/var/lib/kubelet/kubeadm-flags.env")
```

Comment:
Should this also be taken into account in `migrate_1_3?`?

Reply:
Not sure if it's needed?